<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display_settings_widget</class>
 <widget class="QWidget" name="display_settings_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="driverGroupBox">
     <property name="title">
      <string>Driver</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="videoDriverLabel">
        <property name="text">
         <string>Specify if Ryzom is to be run in OpenGL or Direct3D:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="autoRadioButton">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="openglRadioButton">
        <property name="text">
         <string>OpenGL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="direct3dRadioButton">
        <property name="text">
         <string>Direct3D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="parametersGroupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="windowedLabel">
        <property name="text">
         <string>Specify if Ryzom is to be run in full screen or window mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fullscreenRadioButton">
        <property name="text">
         <string>Full screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="videoModeLabel">
        <property name="text">
         <string>Specify a video mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="videomodeComboBox"/>
      </item>
      <item>
       <widget class="QRadioButton" name="windowedRadioButton">
        <property name="text">
         <string>Windowed</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="windowFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="widthLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="heightLabel">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="heightLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="xLabel">
          <property name="text">
           <string>X position</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="xpositionLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="yLabel">
          <property name="text">
           <string>Y position</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ypositionLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
